terraform/aws/main.tf

```diff
@@ -256,14 +324,47 @@ resource "random_shuffle" "dns_ips" {
 }
 
 # `seed-n` type addresses are dapi endpoints
-resource "aws_route53_record" "masternodes" {
-  zone_id = data.aws_route53_zone.main_domain[0].zone_id
-  name    = "seed-${count.index + 1}.${var.public_network_name}.${var.main_domain}"
-  type    = "A"
-  ttl     = "300"
-  records = random_shuffle.dns_ips.result
 
-  count = length(var.main_domain) > 1 ? 5 : 0
```

Comment:
Can we keep this method of iteration? The new code is less flexible in the number of R53 records created.